docs/mpcie_modules.md

```diff
@@ -162,6 +162,50 @@ operating system.
 
 (1) Refer to quirks, it is known to not work stably on all apu2 board revisions
 
+## Ethernet controllers
+
+### Dual Ethernet Controller Realtek RTL8111
+
+![RTL8111](images/realtek_controller.jpg)
+
+This model works well with apu1 boards. However, it is not seen in apu2 and apu3
+boards with default coreboot settings. To enable it make following steps:
+
+1. On apu2 board make sure to enable option *force mPCIe2 clk* in Payload menu.
+This menu is available during boot process by pressing `F10` button. On apu3 it
+doesn't affect detection by OS. When above option is enabled, you should see
+RTL8111 as PCI device in your OS.
+
+2. Manually download and install missing Realtek firmware. Unfortunately,
+coreboot doesn't support it's firmware during boot process. Therefor, even if
```

Comment:
`Therefore`

I would not say that coreboot doesn't support the Realtek firmware. The case is, that OS is responsible for loading the firmware not coreboot/BIOS. The controller works, however, may have limited functionalities in coreboot without Realtek firmware.

Reply:
I corrected that information